website/settings/defaults.py

```diff
@@ -486,6 +486,7 @@ class CeleryConfig:
         'scripts.generate_sitemap',
         'scripts.premigrate_created_modified',
         'scripts.add_missing_identifiers_to_preprints',
+        'management.commands.deactivate_requested_accounts',
```

Comment:
I think this import needs to be `'osf.management.commands.deactivate_requested_accounts'`.  Can you run the management locally to verify before sending back to CR?